src/main/scala/org/jetbrains/plugins/scala/console/apluscourses/ConsoleExecuteAction.scala

```diff
@@ -0,0 +1,104 @@
+// The reason for this class being in a separate package is that the ConsoleExecuteAction class
+// uses the ScalaLanguageConsole.textSent() method, which is package private. Therefore, in order to
+// call it, we must be in the same package as the console: org.jetbrains.plugins.scala.console.
+
```

Comment:
Should there also be some copyright notice that this code is copied (and modified) from **ScalaConsoleExecuteAction.scala** ?